src/mlpack/tests/imputation_test.cpp

```diff
@@ -48,23 +47,23 @@ BOOST_AUTO_TEST_CASE(DatasetMapperImputerTest)
   arma::mat input;
   MissingPolicy policy({"a"});
   DatasetMapper<MissingPolicy> info(policy);
-  BOOST_REQUIRE(data::Load("test_file.csv", input, info) == true);
+  REQUIRE(data::Load("test_file.csv", input, info) == true);
 
   // row and column test.
-  BOOST_REQUIRE_EQUAL(input.n_rows, 3);
-  BOOST_REQUIRE_EQUAL(input.n_cols, 3);
+  REQUIRE(input.n_rows == 3);
+  REQUIRE(input.n_cols == 3);
 
   // Load check
   // MissingPolicy should convert strings to nans.
-  BOOST_REQUIRE(std::isnan(input(0, 0)) == true);
-  BOOST_REQUIRE_CLOSE(input(0, 1), 5.0, 1e-5);
-  BOOST_REQUIRE_CLOSE(input(0, 2), 8.0, 1e-5);
-  BOOST_REQUIRE_CLOSE(input(1, 0), 2.0, 1e-5);
-  BOOST_REQUIRE_CLOSE(input(1, 1), 6.0, 1e-5);
-  BOOST_REQUIRE_CLOSE(input(1, 2), 9.0, 1e-5);
-  BOOST_REQUIRE_CLOSE(input(2, 0), 3.0, 1e-5);
-  BOOST_REQUIRE(std::isnan(input(2, 1)) == true);
-  BOOST_REQUIRE_CLOSE(input(2, 2), 10.0, 1e-5);
+  REQUIRE(std::isnan(input(0, 0)) == true);
+  REQUIRE(input(0, 1) == Approx(5.0).epsilon(1e-5 / 100));
+  REQUIRE(input(0, 2) == Approx(8.0).epsilon(1e-5 / 100));
+  REQUIRE(input(1, 0) == Approx(2.0).epsilon(1e-5 / 100));
+  REQUIRE(input(1, 1) == Approx(6.0).epsilon(1e-5 / 100));
+  REQUIRE(input(1, 2) == Approx(9.0).epsilon(1e-5 / 100));
+  REQUIRE(input(2, 0) == Approx(3.0).epsilon(1e-5 / 100));
+  REQUIRE(std::isnan(input(2, 1)) == true);
+  REQUIRE(input(2, 2) == Approx(10.0).epsilon(1e-5 / 100));
```

Comment:
Can you make these `1e-7` instead of `1e-5 / 100` please? :)  (This happens in a few places throughout.)